src/routes/zip.ts

```diff
@@ -12,18 +13,14 @@ export const router = express.Router();
  * Request zipping of files. Require directory:string and files:string[] in the request body
  */
 router.post("/", (req, res) => {
-  console.log("Request body : ", req.body);
   const { hasAccess, statusCode, error, directory, fileNames } = hasFileAccess(
     req,
     req.body.directory,
     req.body.files
   );
-  console.log("Has access  : ", hasAccess);
-  console.log("Status code : ", statusCode);
-  console.log("Error       : ", error);
-  console.log("Directory   : ", directory);
-  console.log("File names  : ", fileNames);
+
   if (!hasAccess) {
+    logger.logError("Error       : ", { statusCode, error });
```

Comment:
Is the extra space intentional?

Reply:
Remove the spacing. It made sense while I was debugging. Not anymore